src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier supplier = new ColorSupplier();
+        int number = random.nextInt(100);
```

Comment:
All magic numbers in your code should be constants. You should define a constant for the number 100 with a clear name that indicates its purpose.